projects/ad9213_evb/common/ad9213_evb_fmc.txt

```diff
@@ -0,0 +1,72 @@
+FMC_pin  FMC_port         Schematic_name           System_top_name    IOSTANDARD   Termination
+
```

Comment:
Would it be better to remove the second line since this will make the first line from the .xdc to be empty? (when the script is run)